README.md

```diff
@@ -1144,7 +1144,7 @@ func main() {
 	r := gin.Default()
 
 	// You can also use your own secure json prefix
-	// r.SecureJsonPrefix(")]}',\n")
+	// r.SecureJSONPrefix(")]}',\n")
```

Comment:
We can't change all methods. It will break down the new version so just add rule in golangci-lint yaml file.